Conformity/custom-rule/examples/todo

```diff
@@ -0,0 +1 @@
+populate this folder
```

Comment:
If you not going to add anything until the end of the PR, you can remove this and re add again later

Reply:
This is removed.